apps/console/src/features/users/components/user-roles-edit.tsx

```diff
@@ -288,11 +288,7 @@ export const UserRolesList: FunctionComponent<UserRolesPropsInterface> = (
                 ],
                 "Operations": [{
                     "op": "remove",
-                    "value": {
-                        "users": [{
-                            "value": user.id
-                        }]
-                    }
+                    "path":"users[value eq " + user.id + "]"
```

Comment:
Template strings would be better here IMO.

```suggestion
                    "path": "users[value eq " + user.id + "]"
```